mne/io/eeglab/eeglab.py

```diff
@@ -385,8 +385,8 @@ def read_epochs_eeglab(
 
     Returns
     -------
-    epochs : instance of Epochs
-        The epochs.
+    EpochsEEGLAB : instance of BaseEpochs
+        The epochs EpochsEEGLAB (subclass of BaseEpochs).
```

Comment:
i don't think this technical description is helpful here, i would revert this line

Reply:
Thx for the feedback, the redundant description has been removed